tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -275,8 +310,8 @@ install_kernel_tdx_experimental() {
 #Install sev kernel asset
 install_kernel_sev() {
 	info "build sev kernel"
-	local kernel_url="$(yq r $versions_yaml assets.kernel.sev.url)"
-	
+	local kernel_url="$(get_from_kata_deps assets.kernel.sev.url)"
```

Comment:
This should have been fixed as part of 0a6f0ab3cbdce05d686ebb5045a57511e8c77ce2

Reply:
Moved so that correct commit has fix.